Code/LauncherUnified/Launcher.cpp

```diff
@@ -649,17 +509,10 @@ namespace O3DELauncher
 
         // Create CrySystem.
     #if !defined(AZ_MONOLITHIC_BUILD)
-        AZStd::unique_ptr<DynamicModuleHandle> crySystemLibrary;
-        PFNCREATESYSTEMINTERFACE CreateSystemInterface = nullptr;
-
-        crySystemLibrary = DynamicModuleHandle::Create("CrySystem");
-        if (crySystemLibrary->Load(false))
+        AZStd::unique_ptr<CrySystemModuleHandle> crySystemLibrary = CrySystemModuleHandle::Create();
+        if (crySystemLibrary->Load())
         {
-            CreateSystemInterface = crySystemLibrary->GetFunction<PFNCREATESYSTEMINTERFACE>("CreateSystemInterface");
-            if (CreateSystemInterface)
-            {
-                systemInitParams.pSystem = CreateSystemInterface(systemInitParams);
-            }
+            systemInitParams.pSystem = crySystemLibrary->CreateSystemInterface(systemInitParams);
```

Comment:
What was wrong with this code?

It seems like we could just use the `DynmicModuleHandle` directly as it was being used before